src/backends/webgpu/README.md

```diff
@@ -0,0 +1,13 @@
+This is an experimental backend.
+
+```
+    $ yarn # to install dependencies
+    $ yarn test
+```
+
+# To run the test suite:
+The `$CHROME_BIN` environment variable must be set to the location of a custom Chromium build. 
+
+e.g. in `~/.bash_profile`:
+
+`export CHROME_BIN=$HOME/Documents/PROJECTS/tfjs-core-wrapper/Chromium.app/Contents/MacOS/Chromium`
```

Comment:
Did you find a way to pass in the --enable-unsafe-webgpu flag this way?

Reply:
Yep - https://github.com/tensorflow/tfjs-core/pull/1676/files#diff-dfbfae59ecf2ebd486cbf50b4b4ab0e5R44

Reply:
Aha, awesome.